.github/workflows/camunda-platform-release.yml

```
@@ -252,41 +252,32 @@ jobs:
 
-          # Right now we only support patch releases for generating the changelog
-          # as it is the easiest to automate and the most repetitive work to do on a release
-          if [[ "${{ inputs.releaseVersion }}" =~ ^8\.[1-9][0-9]*\.[1-9][0-9]*$ ]]
-          then
-            # Next, add the release labels to the release's issues, specifying the previous and current release in place of ZCL_FROM_REV and ZCL_TARGET_REV, respectively.
-            #
-            # ZCL_TARGET_REV should be replaced with the tag name for the version you are releasing, and
-            # ZCL_FROM_REV should be replaced as the tag name for the previous version, based on the release type:
-            #
-            # PATCH: the tag for the previous patch version on the same minor branch. e.g. if you're releasing 1.2.3, then ZCL_FROM_REV=1.2.2.
+          # Add the release labels to the release's issues, specifying the previous and current release in place of ZCL_FROM_REV and ZCL_TARGET_REV, respectively.
+          #
+          # ZCL_TARGET_REV should be replaced with the tag name for the version you are releasing, and
+          # ZCL_FROM_REV should be replaced as the tag name for the previous version, based on the release type
 
-            # To find the previous patch version we extract the patch version and subtract by one
-            patchVersion=$(echo "${{ inputs.releaseVersion }}" | sed 's/8\.[1-9][0-9]*\.//')
-            majorMinor=$(echo "${{ inputs.releaseVersion }}" | sed 's/\.[1-9][0-9]*$//')
-            ZCL_FROM_REV="$majorMinor".$(( patchVersion - 1 ))
-            ZCL_TARGET_REV="${{ inputs.releaseVersion }}"
+          # prev_tag.sh given a current release version returns current release type and a corresponding previous version,
+          # E.g. for input "8.8.0-alpha1-rc1" it returns "RC 8.7.0"
+          # As we are interested only in previous version, we use awk '{print $2}' to get it's value
+          ZCL_FROM_REV=$(./.github/scripts/prev_tag.sh "${{ inputs.releaseVersion }}" | awk '{print $2}')
+          ZCL_TARGET_REV="${{ inputs.releaseVersion }}"
 
-            # The following command will add labels to the issues on GitHub.
-            # You can verify this step by looking at closed issues. They should now be tagged with the release.
-            ./zcl add-labels \
-            --token=${{ secrets.GITHUB_TOKEN }} \
-            --from="$ZCL_FROM_REV" \
-            --target="$ZCL_TARGET_REV" \
-            --label="version:$ZCL_TARGET_REV" \
-            --org camunda --repo zeebe
+          # The following command will add labels to the issues on GitHub.
+          # You can verify this step by looking at closed issues. They should now be tagged with the release.
+          ./zcl add-labels \
+          --token=${{ secrets.GITHUB_TOKEN }} \
+          --from="$ZCL_FROM_REV" \
+          --target="$ZCL_TARGET_REV" \
+          --label="version:$ZCL_TARGET_REV" \
+          --org camunda --repo zeebe
```

this is the old name of the repo, needs to be:
```suggestion
          --org camunda --repo camunda
```